.github/workflows/smartbear-master-commit.yaml

```diff
@@ -0,0 +1,37 @@
+name: Commit to Master
+
+# This could be dangerous as it will close tickets based on the string it finds
+# in the commit message.
```

Comment:
Yeah, I'd rather close these tickets manually. There may be multiple commits for an issue.

Reply:
yes, so this optional to use, the use case being this, if you are directly committing against the master branch (no PR, no code reviews) and you want to mark the ticket as "Done" - which thinking more about it, we don't have a proper definition of that transition.

I am going to remove this for now, I think we need to understand better if committing to master directly is a thing, and what transition in jira it should have.